packages/components/src/DateTimePickers/DateTime/Input/Input.component.test.js

```diff
@@ -35,6 +35,9 @@ describe('DateTime.Input', () => {
 	it('should call manager focus callback in input focus', () => {
 		// given
 		const managerValue = {
+			inputManagement: {
```

Comment:
You added dateInputManagement or timeInputManagement, not inputManagement

Reply:
removed this change since it's not necessary. test is about input focus